op-node/rollup/derive/channel_out.go

```diff
@@ -17,6 +17,20 @@ import (
 var ErrNotDepositTx = errors.New("first transaction in block is not a deposit tx")
 var ErrTooManyRLPBytes = errors.New("batch would cause RLP bytes to go over limit")
 
+// ChannelOutApi is the interface implemented by ChannelOut
```

Comment:
In general we should define interfaces at the point of their usage rather than their definition. So this interface should be in the `op-e2e/actions` package. I'd also remove the `API` from it.

Reply:
@clabby not outdated